lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -112,6 +112,7 @@ def purchase(money, payment, options = {})
           end
           r.process do
             post = create_post_for_auth_or_purchase(money, payment, options)
+            post[:card][:processing_method] = 'quick_chip' if quickchip_payment?(payment)
```

Comment:
@ryanbalsdon could you clarify why `quick_chip` processing method has to be set here (instead of here [https://github.com/activemerchant/active_merchant/pull/2625/files#diff-8299be227de5ba32c34c30c42a828c9cR388](url)).  Perhaps a comment might be necessary to prevent someone from accidentally moving this.